src/cli/cli.cpp

```diff
@@ -4097,17 +4097,50 @@ template <> otError Interpreter::Process<Cmd("pskc")>(Arg aArgs[])
     otError error = OT_ERROR_NONE;
     otPskc  pskc;
 
+    /**
+     *@cli pskc
+     *@code
+     *pskc
+     *00000000000000000000000000000000
+     *Done
+     *@endcode
+     *@par api_copy
+     *#otThreadGetPskc
+     */
     if (aArgs[0].IsEmpty())
     {
         otThreadGetPskc(GetInstancePtr(), &pskc);
         OutputBytesLine(pskc.m8);
     }
     else
+
+    /**
+     *@cli pskc <key>
+     *@code
+     *pskc 67c0c203aa0b042bfb5381c47aef4d9e
+     *Done
+     *@endcode
+     *@cparm pskc @ca{<key>}
```

Comment:
cparam

Fix everywhere else in this PR

Also don't use < > in the parameter like this, should be

`@ca{key}`

Fix everywhere else in this PR